Zend/tests/bug46811.phpt

```diff
@@ -9,5 +9,5 @@ var_dump(ini_get("arg_separator.output"));
 ?>
 --EXPECTF--
```

Comment:
In cases like this where the output can now be strictly matched, should this go from EXPECTF to EXPECT?